FreeRTOS_DNS.c

```diff
@@ -468,38 +452,36 @@
  * @brief  fill  pxAddress from pucUDPPayloadBuffer
  *
  * @param [out]  pxAddress
- * @param [in] pucUDPPayloadBuffer
  * @param [in] pcHostName hostname to get its length
  *
  */
-    static void prvFillSockAddress( struct freertos_sockaddr * pxAddress,
-                                    uint8_t * pucUDPPayloadBuffer,
-                                    const char * pcHostName )
+    static BaseType_t prvFillSockAddress( struct freertos_sockaddr * pxAddress,
+                                          const char * pcHostName )
     {
         uint32_t ulIPAddress = 0UL;
+        BaseType_t bHasDot = pdFALSE;
 
         /* Obtain the DNS server address. */
         FreeRTOS_GetAddressConfiguration( NULL, NULL, NULL, &ulIPAddress );
         #if ( ipconfigUSE_LLMNR == 1 )
-            BaseType_t bHasDot = llmnr_has_dot( pcHostName );
-        #endif /* ipconfigUSE_LLMNR == 1 */
+            bHasDot = llmnr_has_dot( pcHostName );
 
-        #if ( ipconfigUSE_LLMNR == 1 )
             if( bHasDot == pdFALSE )
             {
                 /* Use LLMNR addressing. */
-                ( ipCAST_PTR_TO_TYPE_PTR( DNSMessage_t, pucUDPPayloadBuffer ) )->usFlags = 0;
```

Comment:
Why was this statement removed? Do we not need to reset flags?

Reply:
I put it somewhere else
as with the changes at this point, we did not create this buffer yet/